prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -12,6 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+resource "google_monitoring_alert_policy" "ci-absent" {
+  for_each     = var.prow_components
+  project      = var.project
+  display_name = "ci-absent-${each.key}"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "ci-absent-${each.key}"
+
+    condition_monitoring_query_language {
+      duration = "600s"
+      query    = <<-EOT
+      fetch k8s_container
+      | metric 'kubernetes.io/container/restart_count'
+      | filter
+          (resource.container_name == '${each.key}' && resource.namespace_name == '${each.value.namespace}')
+      | align rate(1m)
+      | every 1m
+      | group_by [], [row_count: row_count()]
+      | condition val() > 10 '1'
```

Comment:
Does `val()` reference the aggregated row_count field or does this condition apply before the grouping?

Reply:
This seems wrong to me then then. Count is the number of timeseries results, not the number of restarts which is what we should be conditioning on. Rather than selecting aggregating the `count` in the UI, was there an option for aggregating the `sum` instead?

Reply:
I think you're right. count of rows doesn't reflect how many times a pod restarts

Reply:
updated with `delta` aligner